lib/declarations.d.ts

```diff
@@ -691,3 +691,24 @@ interface IFrameworkVersion {
 	 */
 	displayName: string;
 }
+
+/**
+ * Describes WebViewService
+ */
+interface IWebViewService {
+	supportedWebViews: IDictionary<IWebView[]>;
+	getWebView(platform: string, webViewName: string): IWebView;
```

Comment:
Ideally we should have just `getWebViews(...)` method part of the interface and the other two (`getWebVew(...)` and `getWebViewNames(...)`) should be extension methods, because they can be implemented just using the core method. 

This is unfortunately not possible in the current version of TypeScript :sob: